pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -274,24 +270,13 @@ func (b *Build) BuildPod(images BuildPodImages, buildContext BuildContext) (*cor
 		Args: args(
 			[]string{"-layers=/layers", "-analyzed=/layers/analyzed.toml"},
 			func() []string {
-				if !platformAPILessThan07 {
-					return []string{"-run-image=" + runImage}
-				}
 				return []string{}
 			}(),
```

Comment:
This func, along with L276-L278 can be removed. The `args` func just appends a bunch of string slices together, so if we return an empty slice, it's a noop either way.